hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -58,6 +58,14 @@ class AzkabanHelper {
 
   private final static Logger logger = Logging.getLogger(AzkabanHelper);
   static String passwordString = "Password";
+  final static String CONSOLE_EXCEPTION_MESSAGE = """Cannot access the system console. Be sure to pass --no-daemon in your command. For more info,
+                     |refer to https://github.com/linkedin/linkedin-gradle-plugin-for-apache-hadoop/wiki/Azkaban-Features#password-masking.
+                     |
+                     |At LinkedIn, you must pass --no-daemon and explicitly set the JAVA_HOME environment variable to
+                     |the Java version specified in your product-spec. GRADLE_OPTS environment variable should also be set to
+                     |explicitly specify Xmx for Gradle 5 and newer versions, which set a very low default memory threshold for the launcher
+                     |VM; for example: "export GRADLE_OPTS="-Xmx1024m -XX:MaxMetaspaceSize=256m". Refer to the 'Password Masking' section of
```

Comment:
I think we should encourage setting `org.gradle.jvmargs` in `gradle.properties` instead as that seems more self-contained than setting an environment variable every time.

Reply:
I'll mention both options.

Reply:
@erwa Setting `org.gradle.jvmargs` in `gradle.properties` is "too late". The env var `GRADLE_OPTS` is required to give the launcher VM enough heap to allow builds to run there. When `org.gradle.jvmargs` is set with Gradle 5, you'll still get a single-use daemon spawned in a separate process and that's what breaks console input.